frontend/coprs_frontend/config/copr.conf

```diff
@@ -160,6 +160,15 @@ HIDE_IMPORT_LOG_AFTER_DAYS = 14
 # OIDC_SCOPES = "" # e.g. "openid username profile email"
 # OIDC_TOKEN_AUTH_METHOD="client_secret_post" # possible: client_secret_post, client_secret_basic, none
 
+# Copr expects the OpenID provider to provide unique usernames.  It
+# assumes Ipsilon-style UserInfo by default, which satisfies this
+# requirement; however, if you know that your OID provider uses unique
+# values for preferred_username, you can set this value to True to
+# enable Copr to use non-Ipsilon OID providers.
+# It is set to False by default since the OpenID Connect standard
+# forbids assuming that preferred usernames are unique.
+# OIDC_UNIQUE_PREFERRED_USERNAMES = False
```

Comment:
Can you please add this option to `frontend/coprs_frontend/coprs/config.py` as well?

Reply:
Sure